Resources/Textures/_NF/Objects/Consumable/Food/sushi.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "carpsushi, carpsushislice created by gentlebutter (Discord), avocadosushi, avocadosushislice edited by dustylens (GitHub)",
+  "copyright": "carpsushi, carpsushislice created by gentlebutter (Discord), freshsushi(s), avocadosushi, avocadosushislice edited by dustylens (GitHub)",
```

```suggestion
  "copyright": "carpsushi, carpsushislice created by gentlebutter (Discord), freshsushi(slice), avocadosushi(slice) edited by dustylens (GitHub)",
```
```suggestion
  "copyright": "carpsushi, carpsushislice created by gentlebutter (Discord), freshsushi*, avocadosushi* edited by dustylens (GitHub)",
```
Not sure if this was what you were going for, but I've used globs before, makes sense to me here.